src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,157 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Represents the class that interfaces with the user
+ */
+public class Ui {
+
+    private Scanner inputScanner;
+
+    public Ui() {
+        inputScanner = new Scanner(System.in);
+    }
+
+    /**
+     * Prints welcome message
+     */
+    public void welcome() {
+        String logo = " ____        _        \n"
```

Comment:
Can consider using a constant variable instead since the variable does not appear to need changes throughout the application.